### DISCUSSION TITLE
'invalid deadline 5 at epoch 42020, expected 4'  what does this error mean

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/lotus#7056] [Storage Provider]
This means that you have timed out doing a window post. The reasons for the timeout may be: a problem with your graphics card or not having a graphics card, a transient failure of the miner when accessing storage, such as a poor network when accessing network shared storage, and you need to check these possible reasons to avoid missing a window post next time. 
`lotus-miner proving check [command options] <deadlineIdx>` to check the sector status

note: this discussion  is selected from slack qustions

**Comment by johnwick147** [comment]
you could try getting storx 
http://storx.tech

**Reply by de-authority** [reply]
Please do not spam here, this may result in you being blocked
